src/components/core/layout.css

```diff
@@ -425,7 +425,7 @@ table {
   /* font-size: 0.9rem; */
   /* line-height: 1.2rem; */
   border-collapse: collapse;
-  width: 100%;
+  width: 200%;
```

Comment:
Why this ?

Reply:
I cannot find another way to fix the table width